entrypoint.sh

```diff
@@ -20,6 +22,18 @@ events {
 EOF
 
 if [ "$PROTOCOL" = "HTTP" ]; then
+
+cat <<EOF >/etc/nginx/directory_proxy.conf
+proxy_pass http://${UPSTREAM}:${UPSTREAM_PORT};
+proxy_set_header Host \$host;
+proxy_set_header X-Forwarded-For \$remote_addr;
+error_page 403 405 414 416 500 501 502 503 504 ${ERROR_PAGE};
```

Comment:
does this mean we show a page defined by nginx for these status codes? I prefer django to handle `500, 403` because they can be informative for the user (click here to contact us).